frontend/src/components/Course/Grading/NeedHelp.tsx

```diff
@@ -0,0 +1,39 @@
+const NeedHelp = () => {
+    return (
+        <div className="bg-gradient-to-r from-blue-300 to-blue-500 text-white p-8 rounded-lg my-4 w-full text-justify">
+            <h2 className="text-3xl font-bold mb-6">Welcome to the Arbitre Grading System!</h2>
+            <p className="text-lg mb-6">
+                With our incredible application, you can quickly and easily obtain the grade for each session and exercise.
+                Follow these simple steps to get started:
+            </p>
+            <div className="mb-6">
+                <h3 className="text-2xl font-semibold mb-2">Step 1: Complete the Evaluation Grid</h3>
+                <p>
+                    The grid is organized by sessions, where you will find the exercises to evaluate.
+                    Simply enter the grade that each exercise deserves.
```

Comment:
I think this part could be more clear about what a "grade" means.
For example : 
```suggestion
                   Enter the amount of points that the exercice will be graded on.
```